### PR TITLE
Adding Serbian latin translations

### PR DESCRIPTION
**Pull request by stalker314314**
Serbian latin (sr-Latn) is generated automatically from Serbian (sr) translation. Also changed some wording in original (Serbian) translation.

### CODE REVIEW

**Comment by ykzts** [comment]
I don't know the advantage of adding Serbian latin. Is not Serbian alone enough?

**Comment by stalker314314** [comment]
I am not going to go into constitutional/political/regional/cultural reasons for having both scripts, you can read on [wiki more](https://en.wikipedia.org/wiki/Serbian_language#Writing_system), I guess. It is also defined in [RFC5646](https://tools.ietf.org/html/rfc5646) from [here](https://github.com/tootsuite/documentation/blob/master/Contributing-to-Mastodon/Translating.md#declaring-the-language) and it is in other [projects](https://l10n.gnome.org/teams/sr/). Beauty of it is that Cyrillic version can be automatically converted to latin (I can share script, if needed), and that's why all translators favor Cyrillic to be translated first. I think there is no reason to not have both, but if you don't want to - it's your call.

File: app/helpers/settings_helper.rb [hunk 1]
```diff
@@ -30,6 +30,7 @@ module SettingsHelper
     ru: 'Русский',
     sk: 'Slovensky',
     sr: 'Српски',
+    'sr-Latn': 'Srpski (latinica)'
```
Comment (ykzts) [inline app/helpers/settings_helper.rb]: add comma.

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/helpers/settings_helper.rb (hunk 1 above)
Comment (stalker314314) [reply app/helpers/settings_helper.rb]: Oops, sorry, fixed (same commit, squashed)

**Review by ykzts — APPROVED** [review]

**Comment by stalker314314** [comment]
Thanks (again!)

**Pull request merged**